src/go/util/httptest_util.go

```diff
@@ -71,3 +73,18 @@ func InitMockServerFromPathResp(pathResp map[string]string) *httptest.Server {
 		w.WriteHeader(http.StatusNotFound)
 	}))
 }
+
+// JsonEqual compares two JSON strings after normalizing them.
+func JsonEqual(x, y *string) bool {
```

Comment:
If these are just being used for tests, why pass a pointer? Efficiency doesn't matter, just let this be a copy.

Also maybe return `error` instead of just a bool here. The error can be something like:
```
JSON differ: got: %v \n want: %v
```

Then each test can just output the error instead of duplicating this code.

Reply:
Maybe we can use this 
https://gist.github.com/turtlemonvh/e4f7404e28387fadb8ad275a99596f67

Reply:
Also if you end up returning an `error`, it will be printed in normalized format without having to modify the source.